docs/_static/css/admonition_color_contrast.css

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
The CSS files adjust the styles for the online docs, including for font colors.  I copied these two `.css` files over directly from PlasmaPy.  Some of the style settings are for web accessibility (including to make sure that there's enough color contrast between links & the background, etc.).